pkg/pillar/Dockerfile.in

```diff
@@ -56,7 +56,7 @@ RUN apk add --no-cache \
     coreutils dmidecode libbz2 libuuid ipset       \
     curl radvd ethtool \
     util-linux e2fsprogs libcrypto1.0 xorriso qemu-img \
-    jq e2fsprogs-extra keyutils ca-certificates
+    jq e2fsprogs-extra keyutils ca-certificates smartmontools
```

Comment:
This needs to be moved to storage-init

Reply:
@rvs in the future we might want to extract changes to the smart counters (errors, temp, etc) on a running system.
In such a future would it make sense to have storage-init run a periodic dump of smartctl >/run/something, or does storage-init have to terminate all of its work since it is an onboot container?

Reply:
Moved it to storage-init.
But as @nordmark mentioned, if we want to dump SMART data periodically, then we might not be able to do it from storage-init.

Reply:
Agreed -- but then I'll still advocate running this in a separate, dedicated container publishing this info back to EVE (which reminds me that we need to make it easy enough for things like scripting, etc.).